### PR TITLE
Enable SelectExpandTest on Asp.NET Core

### PR DESCRIPTION
**Pull request by xuzhg**
### Issues

* Enable SelectExpandTest on Asp.NET Core

### Description

* Enable SelectExpandTest on Asp.NET Core

### Checklist (Uncheck if it is not completed)

- [ ] *Test cases added*
- [ ] *Build and test with one-click build and test script passed*

### Additional work necessary

*If documentation update is needed, please add "Docs Needed" label to the issue and provide details about the required document change in the issue.*

### CODE REVIEW

File: test/UnitTest/Microsoft.Test.AspNet.OData/SelectExpandTest.cs [hunk 1]
```diff
@@ -301,6 +286,7 @@ public SelectExpandTest()
 
             // Assert
             Assert.NotNull(response);
+
```
Comment (AlanWong-MS) [inline test/UnitTest/Microsoft.Test.AspNet.OData/SelectExpandTest.cs]: Empty space?

File: test/UnitTest/Microsoft.Test.AspNet.OData/SelectExpandTest.cs [hunk 2]
```diff
@@ -319,6 +305,7 @@ public SelectExpandTest()
             HttpResponseMessage response = await GetResponse(uri, AcceptJson);
 
             // Assert
+            string a = response.Content.ReadAsStringAsync().Result;
```
Comment (AlanWong-MS) [inline test/UnitTest/Microsoft.Test.AspNet.OData/SelectExpandTest.cs]: This string does not appear to be used.

File: test/UnitTest/Microsoft.Test.AspNet.OData/SelectExpandTest.cs [hunk 3]
```diff
@@ -686,7 +711,20 @@ public SelectExpandTestCustomerWithAlias GetSelectExpandTestCustomer([FromODataU
         }
     }
 
-    public class NonODataSelectExpandTestCustomersController : ApiController
+#if NETCORE // Use this attribute to help ASP.NET Core to fiter on the action.
```
Comment (AlanWong-MS) [inline test/UnitTest/Microsoft.Test.AspNet.OData/SelectExpandTest.cs]: typo: fiter->filter

File: test/UnitTest/Microsoft.Test.AspNet.OData/SelectExpandTest.cs (hunk 2 above)
Comment (xuzhg) [reply test/UnitTest/Microsoft.Test.AspNet.OData/SelectExpandTest.cs]: Thanks.

**Pull request merged**